src/cares_wrap.cc

```diff
@@ -69,6 +69,19 @@ using v8::Value;
 
 namespace {
 
+inline uint16_t cares_get_16bit(const unsigned char* p) {
```

Comment:
@XadillaX You should feel free to use `ntohs` and `ntohl` here if you like. The functions are already available and do basically the same job.